speech/snippets/transcribe_auto_punctuation_test.py

```diff
@@ -11,23 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 import os
 
 from google.api_core.retry import Retry
 import pytest
 
 import transcribe_auto_punctuation
 
-RESOURCES = os.path.join(os.path.dirname(__file__), "resources")
+_RESOURCES = os.path.join(os.path.dirname(__file__), "resources")
```

Comment:
Please roll back this!

Reply:
I wrote that because in ALL tests, the `resources` folder's variable is defined like that with an underscore.